src/trading/Trading.sol

```diff
@@ -10,4 +10,9 @@ library Trading {
         // FIXME: None is a placeholder to force Bid to equal 1 and Ask to equal 2, since these are their values in the Serpent files and test scripts.  After the Solidity migration is done, we should remove "None" and change Bid to 0 and Ask to 1 everywhere.
         None, Bid, Ask
     }
+    
+    enum TradeDirections {
+        // FIXME: Same as above
+        None, Buying, Selling
```

Comment:
I believe this enumeration is only used in `Trade.sol`, so it should be same/easy to have it be zero indexed now rather than waiting.

Reply:
It's used in makeOrder and takeOrder. It should be really simple once they're converted.